src/app/loadout-builder/process-worker/process.ts

```diff
@@ -233,6 +235,9 @@ export function process(
               const tier = tiers[index];
               const filter = resolvedStatConstraints[index];
               if (!filter.ignored) {
+                if (tier < statRangesFilteredInStatOrder[index].min) {
+                  statRangesFilteredInStatOrder[index].min = tier;
+                }
```

Comment:
It's possible that this is a touch faster:
```suggestion
                const statRange = statRangesFilteredInStatOrder[index];
                if (tier < statRange.min) {
                  statRange.min = tier;
                }
```